### PR TITLE
Pagination of replay files

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
Displays 30 replays per page. Easily changeable

kuva

### CODE REVIEW

File: app/components/FileLoader.js [hunk 1]
```diff
@@ -282,8 +286,17 @@ export default class FileLoader extends Component {
...
+
+    const howManyLeft = (files.length - 1) - fileBeginIndex
+    let lastFileIndex
+    if (howManyLeft > MAX_NUM_OF_FILES_PER_PAGE) {
+      lastFileIndex = fileBeginIndex + MAX_NUM_OF_FILES_PER_PAGE
+    } else {
+      lastFileIndex = fileBeginIndex + howManyLeft
+    }
+
+    const rows = files.slice(fileBeginIndex, lastFileIndex).map(
```
Comment (JLaferri) [inline app/components/FileLoader.js]: What would happen in the case where you're on say, page 5 in a folder, and then you switch to a different folder that has only one page?

Comment (VilleMiekkoja) [reply app/components/FileLoader.js]: Need to check. But my guess is, that the fileloader is then completely reloaded, and so these files and pages are calculated again. But yeah need to check, good point

File: app/components/FileLoader.js [hunk 2]
```diff
@@ -282,8 +286,17 @@ export default class FileLoader extends Component {
...
-    const rows = files.map(
+    const fileBeginIndex = (this.props.store.currentPage - 1) * MAX_NUM_OF_FILES_PER_PAGE
+
+    const howManyLeft = (files.length) - fileBeginIndex
+    let lastFileIndex
+    if (howManyLeft > MAX_NUM_OF_FILES_PER_PAGE) {
+      lastFileIndex = fileBeginIndex + MAX_NUM_OF_FILES_PER_PAGE
+    } else {
+      lastFileIndex = fileBeginIndex + howManyLeft
+    }
```
Comment (fztgreen) [inline app/components/FileLoader.js:297]: Probably should just use a ternary here, more concise
```
lastfileIndex =  fileBeginIndex + ((files.length) - fileBeginIndex) > MAX_NUM_OF_FILES_PER_PAGE ? + MAX_NUM_OF_FILES_PER_PAGE :  howManyLeft
```

**Comment by JLaferri** [comment]
Closing this in favor of #58

**Pull request closed without merging**